code/modules/mob/living/simple_animal/hide_action.dm

```diff
@@ -0,0 +1,20 @@
+/datum/action/innate/animal_hide
```

Comment:
there no good way to incorporate this with the other one?

Reply:
not really, uses totally different layers and stuff

Reply:
Can make the new layer into a variable. Then have the larva one be a subtype of this one. The messages could also be variables if needed

Reply:
it's done